acceptance_tests/features/environment.py

```diff
@@ -10,4 +14,29 @@ def after_all(context):
 
 
 def before_all(context):
-        database_controller.reset_database('resources/database/databasereset.sql')
+    database_controller.reset_rm_database('resources/database/database_reset_rm.sql')
+    database_controller.reset_ras_database()
+    common.signed_in_rops(context)
+    enrolment_setup()
+
+
+def enrolment_setup():
```

Comment:
Not needed for this PR but some better error handling here would make it clearer what fails if we have problems in this script